content/docs/engine-dev/assets/filetypes/texture-support.md

```diff
@@ -0,0 +1,113 @@
+    * For example, if you have `AutomatedTesting\MyTexture.png` and `MyGem\Assets\MyTexture.png`, right now you will only see one of those. With meta files, both will process.
+    * This isn't a unique problem to images, but historically images made more use of this system than other file types.
+
+### What breaks if you move an image?
+
+* Authored materials using the Material Editor reference images via relative path. So if you move an image referenced by a material, that material will break, even if the UUID is stable.
+    * TODO: With this reference as a job dependency, it will behave differently than it does now, which hasn't been tested yet.
+    * Now: the source asset `sponza_mat_curtainblue.material` has a relative path reference to the source asset `"../Textures/curtain_metallic.png"` via a path and not asset ID. If the file is renamed to `bcurtain_metallic.png`, `sponza_mat_curtainblue.material` is not reprocessed, and the product asset `sponza_mat_curtainblue.azmaterial` remains the same, with the same asset hint to the old image and the asset ID of the old image.
+* FBX files and other 3D scene files reference images via absolute or relative path. The scene builder resolves this down to an asset ID for runtime. If you move an image, the FBX file will continue to point at the old file name or location, and will fail to process.
+    * TODO: There's a small chance that, due to lack of non-product dependencies on images, a relocation wouldn't break until later.
+        * Example: `Car.fbx `references `CarBody.png`. The scene builder resolves this to the UUID + sub ID for `CarBody.streamingimage`. Renaming `CarBody.png` to `CarBody1.png`, the sub ID and UUID are stable. `Car.fbx` does not reprocess after this change because it's not a job or source dependency. `Car.fbx`'s product assets referencing `CarBody.png` appear to just work and continue to reference `CarBody1.png` because the product assets haven't changed. Then, when re-exporting `Car.fbx` with a change, the image stops working. Rolling back the change, the image stays broken because the FBX file can no longer be processed as it was before, it's not able to resolve this path reference.
+* UI referencing images with the SimpleAssetReference will break, because the path no longer resolves.
+* Texture Atlas lists and Material Types with hardcoded path references will break, because the path no longer resolves.
+
+## Recommendation
+
+Before enabling image relocation:
+
+* Source asset materials should be updated to use asset IDs instead of relative paths.
+* Scene files should emit job dependencies on referenced images.
+
+There are other potential concerns that are lower priority, and can be worked around:
+
+* UI references to images via SimpleAssetReference
+* Material Types reference images
+
+## Asset Relocation Challenges
+### Materials authored in the material editor have path based references to images

```

Comment:
```suggestion
## Asset Relocation Challenges

### Materials authored in the material editor have path based references to images

```

NIT: To prevent MD parsing/rendering issues, it's advised to have white space new line between all title blocks.